models/issue_label.go

```diff
@@ -98,7 +98,11 @@ func (label *Label) ForegroundColor() template.CSS {
 
 // NewLabels creates new label(s) for a repository.
 func NewLabels(labels ...*Label) error {
-	_, err := x.Insert(labels)
```

Comment:
`x.Insert(labels...)` doesn't compile, since `labels` is of type `[]*Label`, and the argument to `x.Insert(..)` has to be of type `[]interface{}`.

I don't understand why is a transaction needed? From what I can tell, the call to `x.Insert(..)`will just generate one `INSERT` command.